cobra.go

```diff
@@ -205,3 +206,18 @@ func stringInSlice(a string, list []string) bool {
 	}
 	return false
 }
+
+func er(msg interface{}) {
+	if msg != nil {
+		fmt.Println("Error:", msg)
```

Comment:
If this is intended to print out errors, I wonder if we should do something like 
```suggestion
		fmt.Fprintln(os.Stderr, "Error:", msg)
```